### PR TITLE
Feature/42 data api

### PR DESCRIPTION
**Pull request by nirav9spl**
#43 static config.js

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,28 +1,50 @@
 asgiref==3.4.1
+attrs==21.4.0
```
Comment (dedenbangkit) [inline backend/requirements.txt:2]: Clean up these requirements. We only need to install jsmin==3.0.0

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: backend/requirements.txt (hunk 1 above)
Comment (dedenbangkit) [reply backend/requirements.txt]: Can we also remove:
```
asgiref==3.4.1
backports.zoneinfo==0.2.1
pytz==2021.3
sqlparse==0.4.2
mccabe==0.6.1
six==1.16.0
```

Comment (nirav9spl) [reply backend/requirements.txt]: These are support libraries of Django, It will be installed with it anyway

Comment (dedenbangkit) [reply backend/requirements.txt]: Next time when we install a dependencies, the command should be:
```bash
pip install <the-library>
pip freeze | grep <the-library> >> requirements.txt
```

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**